src/components/about/About.js

```diff
@@ -102,7 +102,17 @@ const About = () => {
             </p>
           </div>
           <div className="about-bot-container">
-            <h2>Our Twitter Bot</h2>
+            <h2>
+              Our Twitter Bot
```

Comment:
This was a good catch and is much less misleading than inviting people to "meet" our bot.